languages/python/django-oso/django_oso/auth.py

```diff
@@ -90,8 +90,8 @@ def authorize_model(request, model, *, actor=None, action=None) -> Q:
             filter = Q()
 
         next_filter = partial_to_query_filter(resource_partial, model)
-        if next_filter == Q():
-            return next_filter
+        if next_filter == TRUE_FILTER:
```

Comment:
Are there no cases where we return an empty `Q()` filter? I think that used to happen in some paths, in which case we would want to check if `next_filter` is `TRUE_FILTER` or `Q()`.

Reply:
None that the tests hit, and line 90 above is now the only place we create the empty filter. I thought it would be better to remove all uses to be consistent.